livekit-plugins/livekit-plugins-playai/livekit/plugins/playai/tts.py

```diff
@@ -0,0 +1,313 @@
+from __future__ import annotations
+
+import asyncio
+import logging
+import os
+from dataclasses import dataclass
+
+from livekit import rtc
+from livekit.agents import (
+    DEFAULT_API_CONNECT_OPTIONS,
+    APIConnectionError,
+    APIConnectOptions,
+    tokenize,
+    tts,
+    utils,
+)
+from pyht import AsyncClient as PlayHTAsyncClient  # type: ignore
+from pyht.client import Format, Language, TTSOptions  # type: ignore
+
+from .models import TTSModel
+
+
+@dataclass
+class _TTSOptions:
```

Comment:
Is there a reason to reimplement this class instead of using the imported `TTSOptions` from `pyht` on [this line](https://github.com/livekit/agents/pull/1174/files#diff-dd57d2ed46c52ac07ce07d9ec376392e537c01aef81b937938991438d355d18bR18)?

A number of new options have been added to the class in `pyht` which aren't here, and this may continue to happen.